docs/content/dagster-plus/deployment/azure/blob-compute-logs.mdx

```diff
@@ -0,0 +1,119 @@
+
+```bash
+az aks update --resource-group <resource-group> --name <cluster-name> --enable-workload-identity
+```
+
+Then, we'll create a new managed identity for the AKS agent, and a new service account in our AKS cluster.
+
+```bash
+az identity create --resource-group <resource-group> --name agent-id
+kubectl create serviceaccount dagster-agent-sa --namespace dagster-agent 
+```
+
+Now we need to federate the managed identity with the service account.
+
+```bash
+az identity federated-credential create \                                                                             
+  --name dagster-agent-federated-id \
+  --identity-name agent-id \
+  --resource-group <resource-group> \
+  --issuer $(az aks show -g <resource-group> -n <aks-cluster-name> --query "oidcIssuerProfile.issuerUrl" -otsv) \
+  --subject system:serviceaccount:dagster-agent:dagster-agent-sa
+```
+
+Finally, we'll edit our AKS agent deployment to use the new service account.
+
+```bash
+kubectl edit deployment <your-user-cloud-deployment> -n dagster-agent 
+```
+
+In the deployment manifest, add the following lines:
+
+```yaml
+metadata:
+  ...
+  labels:
+    ...
+    azure.workload.identity/use: "true"
+spec:
+  ...
+  template:
+    ...
+    spec:
+      ...
+      serviceAccountName: dagster-agent-sa
+```
+
+If everything is set up correctly, you should be able to run the following command and see an access token returned:
+```bash
+kubectl exec -n dagster-agent -it <pod-in-cluster> -- bash
+# in the pod
+curl -H "Metadata:true" "http://169.254.169.254/metadata/identity/oauth2/token?resource=https://storage.azure.com/"
+```
+
+## Step 2: Configure Dagster to use Azure Blob Storage
+
+Now, you need to update the helm values to use Azure Blob Storage for logs. You can do this by editing the `values.yaml` file for your user-cloud deployment.
+
+Pull down the current values for your deployment:
+
+```bash
+helm get values user-cloud > current-values.yaml
+```
+
+Then, edit the `current-values.yaml` file to include the following lines:
+
+```yaml
+computeLogs:
+  enabled: true
+  custom:
+    module: dagster_azure.blob.compute_log_manager
+    class: AzureBlobComputeLogManager
+    config:
+      storage_account: chriscomplogmngr
+      container: mycontainer
+      default_azure_credential:
+        exclude_environment_credential: false
+      prefix: dagster-logs-
+      local_dir: "/tmp/cool"
+      upload_interval: 30
+```
+    
+Finally, update your deployment with the new values:
+    
+```bash
+helm upgrade user-cloud dagster-cloud/dagster-cloud-agent -n dagster-agent -f current-values.yaml
+```
+
+## Step 3: Verify logs are being written to Azure Blob Storage
+
+You should be able to kick off a run of `all_assets_job`, and when you go to the stdout/stderr window of the run page, you should see a log file that directs you to the Azure Blob Storage container.
```

Comment:
That's assuming that the user has faithfully followed previous steps without deviation and is using the quickstart_etl module. Might be a bit confusing to some. 

Maybe some wording like, "It's time to kick off a run to test your logging setup. If your code locations still include the quickstart_etl module from step 2 then the `all_assets_job` will be present, otherwise any job of your own that emits some long will do"